src/utilities/Api.js

```diff
@@ -0,0 +1,62 @@
+const API_URL = import.meta.env.VITE_API_URL;
+
+// fetch for categories
+export const getCategories = async () => {
```

Comment:
Where will no get categories. Change this to `login` cause, if we were auth we will receive an api response with the current data. 

Also create another file called "auth.ts". In there paste the login route. Its better to separate contexts.